openreview/tools.py

```diff
@@ -703,3 +703,12 @@ def post_submission_groups(client, conference_id, submission_invite, chairs):
             readers=[conference_id, chairs],
             signatories=[]))
 
+def get_all_venues(client):
```

Comment:
Could you add a test for this?
It should get the venues from the dev site and shouldn't be an empty result or an error

Reply:
Test added in a new file tests/test_tools.py